packages/flutter/lib/src/animation/ticker.dart

```diff
@@ -74,6 +74,6 @@ class Ticker {
   void _scheduleTick() {
     assert(isTicking);
     assert(_animationId == null);
-    _animationId = scheduler.requestAnimationFrame(_tick);
+    _animationId = scheduler.scheduleFrameCallback(_tick);
```

Comment:
If you're in this code anyway, you might want to have this function return an object with a `cancel` method instead of returning an int.  The int return value was just cargo-culted from the web.